configuration/connect-to-ms-exchange.md

```diff
@@ -0,0 +1,233 @@
+# How to connect to Microsoft Exchange with Qubes OS
+
+## Introduction
+This Howto describes how you can connect to your Exchange Server with Qubes OS.
+It is based on using a Exchange-to-IMAP-Gateway which is named Davmail.
+
+The howto covers:
+
+- Installation/Configuration of Davmail
+- Configuration to use Thunderbird
+- Configuration to use neomutt
+
+the following VMs will be mentioned in this howto:
+
+- t-mail = TemplateVM
+- my-vault = ApppVM which is used for SplitGPG
+- my-workmail = AppVM
+
+TODO:
+
+- Howto use SplitGPG with neomutt
+- Howto work with attachments (PDF/Pictures) and HTML mails in neomutt
+
+## Davmail
+
+install required packages for Davmail
+
+`sudo dnf install java-9-openjdk`
+
+Download Davmail in an AppVM and qvm-copy-to-vm it over to the mail template vm
+in the AppVM which has internet access:
+
+`wget https://kent.dl.sourceforge.net/project/davmail/davmail/4.8.3/davmail-linux-x86_64-4.8.3-2554.tgz`
```

Comment:
Is there any way to verify authenticity of the archive? Some signature? Checksum on a page hosted somewhere else (independently of sourceforge.net)? It would be good to include some method here. If there are none, then add a warning about that.

Reply:
Yes, thank you for pointing this out.
I have asked in the Davmail Mailinglist and they provide Checksums on the same sourceforge page where you download the packages.
For example for the current version:
https://sourceforge.net/projects/davmail/files/davmail/4.8.3/checksums.md5/download